lib/jekyll-compose/file_editor.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+#
+# This class is aimed to open created file in selected editor.
+# To use this feature specify at Jekyll config:
+#
+# ```
+#  jekyll_compose:
+#    auto_open: true
+# ```
+#
+# And make sure, that you have JEKYLL_EDITOR or EDITOR env variables set up.
+# This will allow to open file in vim editor automatically.
+
+module Jekyll
+  module Compose
+    class FileEditor
+      attr_reader :file_creator
+
+      def initialize(file_creator)
+        @file_creator = file_creator
```

Comment:
Since we're not using this object for anything else, I wonder if we could just make `open_editor` a static method that takes a `filepath` and opens that file in the editor.

I feel like that might make the code a bit easier to read. What do you think?

Reply:
I suppose then it will pollute `Jekyll::Commands::Post` class with other static methods, that are currently located at `FileEditor`.
But if you insist we can move it there.

Reply:
I don't understand. I think we should keep this `FileEditor` class, but basically just make the methods static.

Reply:
ah, ok! Understood 👍

Reply:
Fixed!